source/rst/continuous_time/seir_model_sde.rst

```diff
@@ -0,0 +1,913 @@
+
+In this section, we will look at example of a diffusion process.
+
+
+Shocks to Transmission Rates
+------------------------------
+
+Consider that the effective transmission rate :math:`R(t)` could depend on degrees of randomness in behavior and implementation.  For example,
+
+* Misinformation on Facebook spreading non-uniformly
+* Large political rallies, elections, or protests
+* Deviations in the implementation and timing of lockdown policy within demographics, locations, or businesses within the system.
+* Aggregate shocks in opening/closing industries
+
+To implement this, we will add on a diffusion term to :eq:`Rode` with an instantaneous volatility of :math:`\zeta \sqrt{R}`.  The scaling by the :math:`\sqrt{R}` ensures that the process can never go negative since the variance converges to zero as R goes to zero.
+
+The notation for this `SDE <https://en.wikipedia.org/wiki/Stochastic_differential_equation#Use_in_probability_and_mathematical_finance>`__ is then
+
+.. math::
+    \begin{aligned} 
+    d R_t &= \eta (B_t - R_t) dt + \zeta \sqrt{R_t} dW_t\\
+    \end{aligned}
+    :label: Rsde
+
+where :math:`W` is standard Brownian motion (i.e a `Weiner Process <https://en.wikipedia.org/wiki/Wiener_process>`__.  This equation is used in the `Cox-Ingersoll-Ross <https://en.wikipedia.org/wiki/Cox%E2%80%93Ingersoll%E2%80%93Ross_model>`__ and `Heston <https://en.wikipedia.org/wiki/Heston_model>`__ models of interest rates and stochastic volatility.
+
+Heuristically, if :math:`\zeta = 0`, divide this equation by :math:`dt` and it nests the original ODE in :eq:`Rode`.
+
+The general form of the SDE with these sorts of continuous-shocks is an extension of our :eq:`dfcv` definition .
+
+.. math::
+    \begin{aligned} 
+    d x_t &= F(x_t,t;p)dt + G(x_t,t;p) dW
+    \end{aligned}  
+
+Here, it is convenient to :math:`d W` with the same dimension as :math:`x` where we can use the matrix :math:`G(x,t;p)` to associate the shocks with the appropriate :math:`x`.
+
+As the shock only effects :math:`dR`, which is the 5th equation, define the matrix as
+
+.. math::
+    \begin{aligned}
+    diag(G) &:= \begin{bmatrix} 0 & 0 & 0 & 0 & \zeta \sqrt{R} & 0 & 0 & 0 \end{bmatrix}
+    \end{aligned}
+
+
+Since these are additive shocks, we will not need to modify the :math:`F` from our equation.
+
+First create a new settings generator, and then define a `SDEProblem <https://docs.sciml.ai/stable/tutorials/sde_example/#Example-2:-Systems-of-SDEs-with-Diagonal-Noise-1>`__  with Diagonal Noise. 
+
+We solve the problem with the `SRA <https://docs.sciml.ai/stable/solvers/sde_solve/#Full-List-of-Methods-1>`__ algorithm (Adaptive strong order 1.5 methods for additive Ito and Stratonovich SDEs)
```

Comment:
this is not an additive noise SDE

Reply:
I see.  This is because of the scaling of the variance?  If so, maybe putting in a note on what the general goto algorithms are for additive vs. other noise couldn't hurt.  Economists (and definetely me!) are utterly ignorant of this stuff.  Everything is homegrown Euler-Maruyama at best.

Reply:
Yes, additive noise is what dW is only multiplied by constants. Here it's just diagonal noise.